ppdet/data/transform/__init__.py

```diff
@@ -16,7 +16,9 @@
 from . import batch_operators
 
 from .operators import *
+from .operator import *
```

Comment:
add TODO to indicate the relationship between operators and operator

Reply:
Done, thanks!